front/src/components/ProfileView/ArtistProfileView.tsx

```diff
@@ -8,17 +9,24 @@ import Biography from './ProfileSections/Biography';
 import MusicProfiles from './ProfileSections/MusicProfiles';
 import Socials from './ProfileSections/Socials';
 import ProfileView from './ProfileView';
+import { ProfileLoadingContext } from '../../pages/Profile/Profile';
 
 interface IArtistProfileViewProps {
   profile: IArtistProfile;
 }
 
 const ArtistProfileView: React.FC<IArtistProfileViewProps> = ({ profile }) => {
+  const profileLoading = useContext(ProfileLoadingContext);
+
   const getProfileSections = (profile: IArtistProfile): JSX.Element[] => {
     const sections: JSX.Element[] = [];
     isBioSectionAvaiblable(profile.description) &&
       sections.push(
-        <Biography key="bio" content={profile.description as string} />
+        <Biography
+          loading={false}
+          key="bio"
+          content={profile.description as string}
+        />
```

Comment:
Not important, but if you want you can set loading to false by default in the Biography component so you don't need it here